### ISSUE TITLE
Improve About Us Text

### ISSUE THREAD

**Issue by Kavakuo** [LabHive/LabHive#50]
The About Us text needs to be improved. It is currently a wall of text, hard to parse. More headings, bullet points, paragraphs, pictures etc. are required to make it beautiful.

When the text is finished, the following things have to be done:
- [ ] Export text from notion as Markdown
- [ ] Convert Markdown to HTML
- [ ] Replace the HTML in [labshare-client/src/views/UeberUns.vue](https://github.com/Kavakuo/LabHive/blob/master/labshare-client/src/views/UeberUns.vue)

**Comment by frwms** [comment]
Text got overhauled by the communication team and is now shorter and without a separate header.
